edgelet/contrib/config/linux/template.toml

```diff
@@ -0,0 +1,154 @@
+# ==============================================================================
+# Hostname
+# ==============================================================================
+#
+# Comment out the next line to override the default hostname of this device.
+#
+# hostname = "my-device"
+
+
+# ==============================================================================
+# Provisioning
+# ==============================================================================
+#
+# Comment out one block and replace the stub values with yours.
+
+
+## Manual provisioning with connection string
+#
+# [provisioning]
+# source = "manual"
+# connection_string = "HostName=example.azure-devices.net;DeviceId=my-device;SharedAccessKey=YXppb3QtaWRlbnRpdHktc2VydmljZXxhemlvdC1pZGU="
```

Comment:
Remove the connection string, and put something like <IoT Edge device connection string> or soemthinglike that.
Also that key could get caught in scanners.

Reply:
We should already have an exemption for the key from the git submodule days.